packages/components/src/styles/meta/meta.stories.ts

```diff
@@ -35,25 +35,25 @@ export const Default = {
 };
 
 /**
- * Use `sd-meta` modifiers for alternative appearances.
+ * Use the `sd-meta` classes for alternative appearances:
  *
- * - Black is the default
- * - `sd-meta--light`: Neutral-700 can be used to deemphasize text content
+ * - Black is the default color
+ * - `sd-meta--light`: Neutral-700
  */
 
 export const Variants = {
   render: () =>
     html`<div class="flex flex-row gap-12">
-      <span class="sd-meta">Black</span>
-      <span class="sd-meta sd-meta--light">Neutral-700</span>
+      <span class="sd-meta">Default</span>
+      <span class="sd-meta sd-meta--light">Light</span>
     </div>`
 };
 
 /**
- * Use `&--size-*` class for alternative appearances.
+ * Use `sd-meta` classes for alternative appearances:
  *
  * - lg is the default size
```

Comment:
In this case, me and @MartaPintoTeixeira believe it shouldn't be highlighted because it is not a class or a value. Therefore, it serves as a way to differentiate. We think the highlight may mislead the user into believing it is also a class.

Reply:
Updated from lg to base in https://github.com/solid-design-system/solid/pull/1426